sources/install/package_ad.sh

```diff
@@ -1244,4 +1263,6 @@ function package_ad() {
     install_netexec                # Crackmapexec repo
     install_extractbitlockerkeys   # Extract Bitlocker recovery keys from all the computers of the domain
     install_LDAPWordlistHarvester
+    install_freeipscanner
+    # install_scrtdnsdump          # This tool is a fork of adidnsdump (https://github.com/dirkjanm/adidnsdump). We are currently waiting to see if a PR will be made.
```

Comment:
A PR was made then closed by adidnsdump' author: https://github.com/dirkjanm/adidnsdump/pull/15

Reply:
Not sure we can. Fork relies on scapy which is GPLv2, and we are GPLv3. I think we can’t…

Reply:
For now, can we merge freeipscanner and let scrtdnsdump as draft ?

Reply:
yup